Automattic/newspack-plugin
# feat(ia): settings additional brands

### All Submissions:


### Changes proposed in this Pull Request:

Adds Newspack Multibranded Site wizard to Newspack Plugin Settings Wizard. The majority of functionality is merely a port of the Multibranded plugin.

### How to test the changes in this Pull Request:

1. Checkout this branch and build assets.
2. Ensure Newspack Multibranded Site plugin is installed and active.
3. Navigate to _/wp-admin/admin.php?page=newspack-settings#/additional-brands_
4. Ensure Additional Brands UI loads.
![Screenshot 2024-10-21 at 09 14 41](https://github.com/user-attachments/assets/9ce2d7b5-91b9-4f21-82c1-477b1dddd983)
5. Perform Tests. For each test confirm; 1. data loaded is correct 2. Data persists between tab changes and page refreshes:
   - Add New Brand - Click "Add New Brand", input desired configuration and click "Save".
   - Edit an existing brand - Click ![Screenshot 2024-10-21 at 09 22 32](https://github.com/user-attachments/assets/f265d557-9ae3-43e4-98bb-f041e9242508), select "Edit", perform desired edits and click "Save".
   - Delete a brand - Click ![Screenshot 2024-10-21 at 09 22 32](https://github.com/user-attachments/assets/f265d557-9ae3-43e4-98bb-f041e9242508), select "Delete" and confirm.


### Other information:
